### PR TITLE
feat: added support for preview

### PR DESCRIPTION
**Pull request by L2jLiga**
*Description of changes:*
Added static method to Sketch class to add/replace preview image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: models/Sketch/Sketch.js [hunk 1]
```diff
@@ -75,6 +75,13 @@ class Sketch {
       });
   }
 
+  static addPreview(preview) {
+    if (!fs.existsSync(preview)) return;
```
Comment (dbanksdesign) [inline models/Sketch/Sketch.js]: Should we add something to the console to let the user know there was an issue getting the file? Or throw an error?

Comment (L2jLiga) [reply models/Sketch/Sketch.js]: Maybe something like `console.warn('Error occurred while adding preview image, please check that file in place')`?

Comment (dbanksdesign) [reply models/Sketch/Sketch.js]: Yea, what about 'Error occurred while adding preview image, please check the file path' ?

Comment (L2jLiga) [reply models/Sketch/Sketch.js]: Sounds good for me! Will update PR later today

Comment (L2jLiga) [reply models/Sketch/Sketch.js]: Done

**Pull request merged**